ilan/install_ihs.sh

```diff
@@ -79,4 +76,3 @@ function install_version() {
 
 
 install_version
-tar -zcf /host/ihs${VERSION}.tar.gz /opt/IBM/HTTPServer /opt/IBM/WebSphere/Plugins /opt/IBM/WebSphere/Toolbox
```

Comment:
@davidcurrie The installation-manager image is transient anyway it does not get pushed anywhere.  But I did make most of these changes so I can do a build on Windows #20